cmd/root.go

```diff
@@ -21,8 +21,8 @@ var rootCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		version, _ := cmd.Flags().GetBool("version")
 		if version {
-			fmt.Println(VERSION)
-			os.Exit(1)
+			fmt.Fprintln(cmd.OutOrStdout(), VERSION)
```

Comment:
the `cmd.OutOrStdout` is needed by Cobra to be able to use `rootCmd.SetOut(got)` and `rootCmd.SetErr(got)`, isn't it?

Reply:
As far as I know, yes. This added test also checks the versioning format.
If only need to check `true`, https://github.com/gennaro-tedesco/gh-i/pull/25/commits/74c1a9239de179026d482fd9e7205eb4b2f1c84e might be enough.